config/initializers/sentry.rb

```diff
@@ -1,9 +1,16 @@
 # frozen_string_literal: true
 Sentry.init do |config|
   config.environment = HostingEnvironment.environment_name
+  env_var_filters = Regexp.union(ENV.values)
   filter =
     ActiveSupport::ParameterFilter.new(
-      Rails.application.config.filter_parameters
+      Rails.application.config.filter_parameters +
+        [
+          ->(k, v) do
+            return unless k == "value" && k == "title"
+            v.gsub!(env_var_filters, "[FILTERED]") if env_var_filters.match?(v)
+          end
+        ]
```

Comment:
I think this could look a bit nicer if it was a variable and we hoisted it a bit higher.

Actually think this could be `env_var_filter` and the union could be called like `env_var_values`

Just stylistic nit, non-blocking

Reply:
The union returns a Regexp object and it is not necessarily the values anymore hence why i have named it env_var_filters as it contains an array of Regexp's

Reply:
Right; could we still hoist it? If we do, it makes sense to name it something like `env_filter` in which case we should name the Regexp union something more relating to values. Naming things in this context is a bit difficult because the Regexp is a filter, the `ActiveSupport::ParameterFilter` is a filter, and things get confusing.

What do you think of this; at the least, I much prefer putting the proc in its own variable:

```ruby
  env_values = Regexp.union(ENV.values)
  env_filter = ->(k, v) do
    return unless %i[value title].include?(k)
    v.gsub!(env_values, "[FILTERED]") if env_values.match?(v)
  end
  filter =
    ActiveSupport::ParameterFilter.new(
      Rails.application.config.filter_parameters + [env_filter]
    )
```

Reply:
Makes absolute sense. Will effect this change